book/website/ethical-research/data-hazards/dh-intro.md

```diff
@@ -0,0 +1,109 @@
+```
+
+#### Description
+This hazard is appropriate where methodologies are energy-hungry, data-hungry (requiring more and more computation), or use special hardware that requires rare materials. Alternatively, this could also apply to research which outputs high amounts of waste.
+
+#### Examples
+ - Example 1: Wet-lab experiments which create large amounts of single-use plastic waste, for example, pipette tips or petri dishes; or specific chemicals that are harmful to the environment {cite:ps}`Freeland2022`, {cite:ps}`Leak2023`. 
+ 
+ - Example 2: Running big models on High-Performance Clusters can have a big environmental impact with the energy they consume {cite:ps}`alaparthi2020bidirectional`, {cite:ps}`Bender2021`.
+
+#### Safety Precautions
+ - Consider recycling programmes, as well as reducing purchasing and shipping. 
+ - To communicate the scale of the issue to other stakeholders, you may want to [convert units of energy into more relatable units](https://calculator.green-algorithms.org/).
+ - Find out if your cloud provider uses renewable energy.
+ - Consider profiling your code, and rewriting it to use less energy. 
+ - Consider future work that would reduce the need to use increasingly more resources.
+
+---
+
+### But Why Hazard Labels?
+The Data Hazards framework takes inspiration from the visual nature of chemical hazard symbols, communicating to users the potential outcomes and risks of a project. 
+Similarly, to chemical hazards, labelling something as hazardous does not change or reduce its inherent risk.
+However, the visual display of the risk allows the user to make choices based on this knowledge. 
+The user can choose to take the necessary precautions to keep themselves, others and their environment safe. 
+This also applies to the Data Hazard labels, where if we can identify the risks, we can act accordingly and mitigate possible risks.
+
+
+### Where did they come from?
+There were originally six labels, which were identified through reading data ethics papers at [Data Ethics Club](http://dataethicsclub.com): a fortnightly journal club for data science and ethics.
+Through collecting suggestions [on GitHub](), and at conferences and Data Hazards workshops, further labels were suggested and added until we reached the current 11 labels.
+
+### The Hazard labels are versioned
+<!--The Hazard labels are versioned!-->
+The Data Hazard labels are a living project and will continue to change as we gather more suggestions - for this reason, they are versioned using semantic versioning.
+At the time of writing, in `v1.0`, there are 11 Data Hazard labels: you can [explore the Data Hazard labels yourself](https://datahazards.com/labels) on the Data Hazards website.
+
+(er-datahazardsintro-ethicalframeworks)=
+## Ethical Frameworks 
+Ethical frameworks are perspectives useful for reasoning what course of action may provide the most moral outcome [cite].
```

Comment:
This is a complicated sentence that could do with a little bit of revision.

Also looks like a citation is needed in place of `[cite]`?